viadot/tasks/mysql_to_df.py

```diff
@@ -52,7 +52,7 @@ def run(
         if not credentials_secret:
             try:
                 credentials_secret = PrefectSecret("CONVIDERA").run()
-                logger.info("Loaded credentials from Key Vault.")
+                logger.info("Loaded credentials from PrefectSecret.")
             except ValueError:
```

Comment:
Maybe it will be good to add a logger if there is an error in the `try` block. Information that no credentials were found or something like that